modules.js

```diff
@@ -174,16 +174,132 @@ const isModuleHTMLFile = filePath => MODULE_HTML_EXTENSION_REGEX.test(filePath);
  */
 const isModuleCSSFile = filePath => MODULE_CSS_EXTENSION_REGEX.test(filePath);
 
+// Strings to replace in React module files
+const MODULE_STRING_TRANSFORMATIONS = [
+  {
+    regex: /\/\* import global styles \*\//g,
+    string: 'import "./global-samplejsr.css";',
+    fallback: '',
+  },
+  {
+    regex: /\/\* Default config \*\//g,
+    string:
+      'export const defaultModuleConfig = { \n  moduleName: "sample_jsr", \n  version: 0, \n};',
+    fallback: '',
+  },
+];
+/**
+ * createModule() helper - Takes a file and uses the constants above to transform the contents
+ *  @param {string} file - the file being manipulated
+ * @param {object} metaData - an object containing the module's metadata
+ */
+
+const transformFileContents = (file, metaData, getInternalVersion) => {
```

Comment:
Is there an issue here where this function is doing async callbacky things bu there is no returned promise of async/await handling?